src/_nested-list.scss

```diff
@@ -85,21 +85,10 @@ $button: #{$fd-namespace}-button;
     border-bottom: none;
 
     .#{$block}__group-header {
-      @include fd-nested-list-popover-border-top-radius();
```

Comment:
assuming, nested list popover will always have popover header. so border rounding is not needed on group header,